### ISSUE TITLE
bash completion for kompose 

### ISSUE THREAD

**Issue by janetkuo** [kubernetes-incubator/kompose#37]
A nice-to-have feature.

**Comment by ngtuna** [comment]
First commit https://github.com/skippbox/kompose/commit/f09b640e90015dc559af1d6c45d2570940d56b7c

**Comment by sebgoa** [comment]
Havent' tested this, can we close this ticket based on the commit ?

**Comment by ngtuna** [comment]
We still should cover bash completion for flags. The first commit supports subcommands only.

**Comment by sebgoa** [comment]
@cab105 maybe you can take that one as well...

**Comment by cab105** [comment]
@sebgoa Sure I'll take a look at it.

**Comment by cdrage** [comment]
@cab105 @sebgoa I can take this over once #304 is merged, as it has implemented support for bash completion.

**Comment by kadel** [comment]
closing, this was done in #370
